### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [organicmaps/organicmaps#1453] [Q&A] [answered]
Would a Mastodon account for this project be possible?

**Comment by biodranik** [comment]
Probably. Why do you think it is a good idea to support another social network?

**Reply by linux-lukas** [reply]
Non-free social networks, such as Facebook, Instagram, Twitter, etc. are very questionable in terms of data protection. That's why my suggestion would be to have an account on a free social network, like Mastodon, which values data protection. 

This results in two things from my point of view: 
- The project itself would become better known 
- Mastodon as an alternative to non-free networks would be strengthened and also become better known.

**Comment by MXC48-zz** [comment]
> Probably. Why do you think it is a good idea to support another social network?

For people like me who do not have Facebook, twitter etc... By choice but I'm on Mastodon was much better! And then make a free and open-source application to not be tracked but use non-free social networks is not great...

**Reply by biodranik** [reply]
Good point. I'll check it out.

**Reply by linux-lukas** [reply]
Personally, I think a project mastodon account could then help the project in the future to find people from the programming field who can help fix problems.

**Reply by biodranik** [reply]
Which community do you recommend to join?

**Reply by linux-lukas** [reply]
I could imagine `social.tchncs.de` as a Mastodon node?

**Reply by MXC48-zz** [reply]
> Which community do you recommend to join?

Try mastodon.social (https://mastodon.social/about) or find out if there are any for companies or you can host your own instance

**Comment by rtsisyk** [comment]
@biodranik, I was working on it.

**Reply by MXC48-zz** [reply]
Ok thanks

**Comment by ddpasa** [comment]
Mastodon is experiencing explosive growth right now.  More than 600,000 users have signed up within the last week alone: https://bitcoinhackers.org/@mastodonusercount/109376255455025008  Mastodon is especially popular among open source enthusiasts.

As such having a presence on Mastodon would be valuable for Organic Maps.  There are also bots that allow automatic mirroring between Twitter and Mastodon to make this less of an effort.

**Reply by biodranik** [reply]
The main question is now about the server. Who should run/maintain it?

**Reply by stephan75** [reply]
For an account, you dont need a server on your own ... Right?

**Reply by Mixaill** [reply]
Fosstodon is quite popular among OSS (rust-lang, pipewire, python, qgis)

https://fosstodon.org/

**Reply by JacksonChen666** [reply]
> For an account, you dont need a server on your own ... Right?

no

**Reply by hb0nd** [reply]
Fosstodon looks like a good choice 👍

**Reply by ddpasa** [reply]
I agree with above.  This should be minimal effort if you create an account on an existing server (like fosstodon).  You can also create a bot to mirror the content with Twitter or manage it separately.

**Reply by JacksonChen666** [reply]
> I agree with above. This should be minimal effort if you create an account on an existing server (like fosstodon). You can also create a bot to mirror the content with Twitter or manage it separately.

an existing account can be marked as a bot, but there are also things like [cross posters](https://moa.party/) (used by [@marcan@treehouse.systems](https://social.treehouse.systems/@marcan)?) which would negate the need to post to both mastodon and twitter manually.

**Reply by linux-lukas** [reply]
If a crosspoter is being considered, I would strongly recommend mirroring the posts from Mastodon to Twitter and not the other way around.

**Reply by ddpasa** [reply]
> If a crosspoter is being considered, I would strongly recommend mirroring the posts from Mastodon to Twitter and not the other way around.

I agree.

Also Twitter/Mastodon cross-posting is a very active topic these days.  It should not be difficult to set this up in fosstodon or msatodon.social.

**Comment by ddpasa** [comment, marked as answer]
Looks like OrganicMaps now has a Mastodon account on Fosstodon: https://fosstodon.org/@organicmaps (and already >100 followers!)

It looks like we can close this issue.  Please spread the word on the Mastodon account.

**Reply by JacksonChen666** [reply]
ummm, this isn't an issue, this is a discussion.

**Comment by stephan75** [comment]
Fine, thank you forcall efforts! Will follow you, too.

Maybe now add an item to https://wiki.openstreetmap.org/wiki/Mastodon in the OSM wiki?

**Reply by hb0nd** [reply]
done on here and the om wiki page 👍

**Reply by ddpasa** [reply]
Thanks!

**Comment by rtsisyk** [comment]
You asked, we did. @Lukas2112 , could you please mark this issue as resolved?

**Comment by JacksonChen666** [comment]
as repeated, this is not an issue. this is a discussion. a discussion cannot be closed or marked as resolved (this is also not a Q&A discussion where answers are selected), only left dead.

**Reply by rtsisyk** [reply]
Fixed.
